src/crimson/os/seastore/segment_manager/block.cc

```diff
@@ -310,9 +310,12 @@ open_device_ret open_device(
     return seastar::open_file_dma(
       path,
       seastar::open_flags::rw | seastar::open_flags::dsync
-    ).then([=, &path](auto file) {
-      INFO("path={} successful, size={}", path, stat.size);
-      return std::make_pair(file, stat);
+    ).then([=, &path](auto file) mutable {
```

Comment:
`mutable` seems not needed here, we only need to modify the inner-most copied `stat`.

Reply:
This mutable is required otherwise build will fail with error “assignment of member ‘seastar::stat_data::size’ in read-only object”

Reply:
ok

Reply:
It's not a great idea to be mutating implicitly captured variables.  Replace the '=' with the name of the variable in this case.